### PR TITLE
fix key work node_id && node_state

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 
fix issue https://github.com/pingcap/tidb/issues/9910
treat `node_id` and `node_stats` as key work

### What is changed and how it works?
move `pump` `drainer` `node_id` `node_stats` to `NotKeywordToken`

### Check List 

Tests 

 - Manual test (create table, use `pump` `drainer` `node_id` `node_stats` as column name)

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

**Comment by kennytm** [comment]
PTAL @zimulala

File: parser.y [hunk 1]
```diff
@@ -484,15 +482,17 @@ import (
...
 	buckets		"BUCKETS"
 	cancel		"CANCEL"
-	ddl		"DDL"
-	drainer         "DRAINER"
+	ddl		    "DDL"
+	drainer     "DRAINER"
 	jobs		"JOBS"
 	job		    "JOB"
-	pump            "PUMP"
+	node_id		"NODE_ID"
```
Comment (lonng) [inline parser.y]:
```suggestion
	nodeId		"NODE_ID"
```
It's better to keep consistent with others.

File: parser.y [hunk 2]
```diff
@@ -267,7 +265,7 @@ import (
 	zerofill		"ZEROFILL"
 	natural			"NATURAL"
 
-	/* The following tokens belong to UnReservedKeyword. */
+	/* The following tokens belong to UnReservedKeyword. Notice: make sure these tokens are contained in UnReservedKeyword. */
```
Comment (kennytm) [inline parser.y]: 🤔 So turns out this is already checked in `consistent_test.go`

File: parser.y (hunk 1 above)
Comment (zimulala) [reply parser.y]: I think `nodeID` is better.

File: parser.y [hunk 3]
```diff
@@ -484,15 +482,17 @@ import (
...
 
-	/* The following tokens belong to TiDBKeyword. */
+	/* The following tokens belong to TiDBKeyword. Notice: make sure these tokens are contained in TiDBKeyword. */
 	admin		"ADMIN"
 	buckets		"BUCKETS"
 	cancel		"CANCEL"
-	ddl		"DDL"
-	drainer         "DRAINER"
+	ddl		    "DDL"
+	drainer     "DRAINER"
```
Comment (zimulala) [inline parser.y]: Please align.

File: parser.y (hunk 2 above)
Comment (WangXiangUSTC) [reply parser.y]: 🤔but it don't take effect when 'node_id' not in keyword list, I don't know why

**Comment by tiancaiamao** [comment]
LGTM

File: parser.y (hunk 2 above)
Comment (lonng) [reply parser.y]: Only put `node_id` in TiDBKeyword part (or other special parts) can take effect.

**Pull request merged**